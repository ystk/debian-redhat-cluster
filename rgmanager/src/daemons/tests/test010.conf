<?xml version="1.0"?>
<!-- Add NFS export to the mix.  Should have no NEEDSTOPS/NEEDSTARTS
     in the resource tree outputs.  -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.3" monitor_link="yes"/>
    		<fs fstype="ext3" name="mount1" mountpoint="/mnt/cluster" device="/dev/sdb8"/>
		<nfsexport name="Dummy Export"/>
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.3"/>
		<script ref="initscript"/>
		<fs ref="mount1"/>
	</service>
</rm>
</cluster>
