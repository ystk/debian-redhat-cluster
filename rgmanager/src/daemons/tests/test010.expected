=== Resources List ===
Resource type: fs
Instances: 1/1
Agent: fs.sh
Attributes:
  name = mount1 [ primary ]
  mountpoint = /mnt/cluster [ unique required ]
  device = /dev/sdb8 [ unique required ]
  fstype = ext3
  nfslock [ inherit("nfslock") ]

Resource type: ip
Instances: 1/1
Agent: ip.sh
Attributes:
  address = 192.168.1.3 [ primary unique ]
  monitor_link = yes
  nfslock [ inherit("service%nfslock") ]

Resource type: nfsexport
Agent: nfsexport.sh
Attributes:
  name = Dummy Export [ primary ]
  device [ inherit("device") ]
  path [ inherit("mountpoint") ]
  fsid [ inherit("fsid") ]

Resource type: script
Agent: script.sh
Attributes:
  name = initscript [ primary unique ]
  file = /etc/init.d/sshd [ unique required ]
  service_name [ inherit("service%name") ]

Resource type: service
Instances: 1/1
Agent: service.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
service {
  name = "test1";
  fs {
    name = "mount1";
    mountpoint = "/mnt/cluster";
    device = "/dev/sdb8";
    fstype = "ext3";
      }
  ip {
    address = "192.168.1.3";
    monitor_link = "yes";
      }
  script {
    name = "initscript";
    file = "/etc/init.d/sshd";
    service_name = "test1";
  }
}
